python/ray/dag/compiled_dag_node.py

```diff
@@ -37,6 +37,14 @@
 
 MAX_BUFFER_COUNT = MAX_BUFFER_TOTAL_MEMORY // MAX_BUFFER_SIZE
 
+
+class RayDAGArgs:
```

Comment:
Can you use a [namedtuple](https://docs.python.org/3/library/collections.html#collections.namedtuple)? This may be a bit better for performance (classes can be modified and I'm guessing serialization is similarly more expensive).

Reply:
Done